### PR TITLE
made changes to include outputting in mitgcm format

### PR DESCRIPTION
**Pull request by anirban89**
workaround to the problem:

`if ((Lx*10.0**4.0)%(dx*10.0**4.0))/10.0**4.0 !=0.0:
            raise ValueError("Lx is not divisible evenly by dx")`

### CODE REVIEW

File: floater/generators.py [hunk 1]
```diff
@@ -24,9 +24,9 @@ def __init__(self, xlim, ylim, dx=1., dy=1.):
         self.ylim = ylim
         self.Lx = xlim[1] - xlim[0]
         self.Ly = ylim[1] - ylim[0]
-        if (self.Lx % dx) != 0.0:
+        if ((self.Lx*10.0**4.0) % (dx*10.0**4.0))/10.0**4.0 != 0.0:
             raise ValueError("Lx is not divisible evenly by dx")
-        if (self.Ly % dy) != 0.0:
+        if ((self.Ly*10.0**4.0) % (dy*10.0**4.0))/10.0**4.0 != 0.0:
```
Comment (rabernat) [inline floater/generators.py]: Can you help me understand what is happening here?

Comment (rabernat) [inline floater/generators.py]: I guess you are just multiplying everything by a big number...is this a robust solution?

**Comment by rabernat** [comment]
I am happy to merge this so we can move on. I think there might be a better solution, but that can wait.

I am working on writing some tests for the generators object. That is part of the problem--we don't have tests.

File: floater/generators.py (hunk 1 above)
Comment (anirban89) [inline floater/generators.py]: I don't know about robustness. Maybe we can figure this out at a later stage

**Pull request merged**